services/ui-src/src/utils/forms/forms.ts

```diff
@@ -203,3 +204,76 @@ export const flattenFormFields = (formFields: FormField[]): FormField[] => {
   compileFields(formFields);
   return flattenedFields;
 };
+
+// repeatable target population choice list
+const repeatableTargetPopulationChoices = (
+  fields: (FormField | FormLayoutElement)[],
+  choiceList: any
+) => {
+  const createLabel = (field: any) => {
+    if (choiceList.indexOf(field) >= 4) {
```

Comment:
[nit, non-blocking]: would you be open to assigning 4 to a variable so we don't have magic strings and to make it more clear that 4 is the number of "default" target populations?

Reply:
yeah, this code is actually from Nate's work, just also here because of a merge conflict. I should adjust this PR